style.md

```diff
@@ -333,7 +333,7 @@ A single blank line appears:
 
  1. _Between_ consecutive members of a class: properties, constructors, functions, nested classes, etc.
 
-     * **Exception**: A blank line between two consecutive properties (having no other code between them) is optional. Such blank lines are used as needed to create logical groupings of properties and associate properties with their backing property, if present.
+     * **Exception**: A blank line between two consecutive properties (having no other code between them) is optional. Such blank lines are used as needed to create logical groupings of properties and associate properties with their backing property if present.
```

Comment:
This seems incorrect. I believe "if present" is a participle phrase which requires a comma.